substrate/frame/grandpa/src/benchmarking.rs

```diff
@@ -18,9 +18,7 @@
 //! Benchmarks for the GRANDPA pallet.
 
```

Comment:
```suggestion
#![cfg(feature = "runtime-benchmarks")]
```

Please add this before the imports

Reply:
fixed, but placed on line 20 - git's not seeing it

Reply:
I cannot see it, can you put it right here before every import?